packaging/installer/functions.sh

```diff
@@ -933,7 +933,7 @@ portable_add_group() {
   groupname="${1}"
 
   # Check if group exist
-  if cut -d ':' -f 1 < /etc/group | grep "^${groupname}$" 1> /dev/null 2>&1; then
+  if getent group | grep -q "^${groupname}:"; then
     echo >&2 "Group '${groupname}' already exists."
```

Comment:
Hello @Dim-P ,

I do not know environment that does not have `getent` command, but supposing there is this environment, is it not better to create a function that check for the existence of the command and if it fails to use `/etc/group`?

Best regards!

Reply:
I agree, this should ideally check for the availability of `getent` (`command -v getent > /dev/null 2>&1` is sufficient for this check) and fall back to the current behavior if it’s not found.

It shouldn’t ever need to fall back, but we should still write the code to sanely handle things if possible.

Reply:
There are a couple of other places where `getent` is used without prior checks if it's available so I assumed it would be fine, but I have made the requested changes.